commerce-inventory-service/src/main/resources/META-INF/custom-sql/default.xml

```diff
@@ -26,7 +26,7 @@
 			WHERE
 				(Group_.groupId = ?) AND
 				(CIWarehouseItem.sku = ?) AND
-				(CIWarehouse.active_ = 1)
+				(CIWarehouse.active_ = true)
```

Comment:
Are you sure we want `true` rather than `1` here? `1` may be more db agnostic -- unless you're making this change exactly for the opposite reason

Reply:
@ethanbustad the reason for the change is that Postgres is not accepting numeric for boolean but the following: TRUE, 't', 'true', 'y', 'yes', '1'.

Reply:
Got it. Yeah, I think we should try to do it the same way portal does it if possible, otherwise we'll most likely have compatibility issues on some _different_ database. E.g. here: https://github.com/ethanbustad/liferay-portal/blob/eaf0d2ce29141f1ace63a1cb2a12db3b6909ca67/portal-impl/src/custom-sql/portal.xml#L63. Looks like once you pass this token through `CustomSQLUtil`, it'll replace it with the proper boolean value for the current db.

Reply:
Got it. Yeah, I think we should try to do it the same way portal does it if possible, otherwise we'll most likely have compatibility issues on some _different_ database. E.g. here: https://github.com/ethanbustad/liferay-portal/blob/eaf0d2ce29141f1ace63a1cb2a12db3b6909ca67/portal-impl/src/custom-sql/portal.xml#L63. Looks like once you pass this token through `CustomSQLUtil`, it'll replace it with the proper boolean value for the current db.

Reply:
Hi @ethanbustad changed the query according to the portal standards